frontend/src/components/FavoriteButton.scss

```diff
@@ -0,0 +1,12 @@
+.favorite-button {
```

Comment:
Can we document this as an override provided by UX and we should talk with PF about it. Also if you could start that conversation I would appreciate it.